slack.js

```diff
@@ -0,0 +1,266 @@
+//------------------------------------------------------------------------------
+// Bot / Slack related helpers
+//------------------------------------------------------------------------------
+var Botkit = require('botkit');
```

Comment:
I dont actually see a use of Botkit in slackjs.

The reference is passed in from the caller.

I believe this can be removed.